src/Chase/PossibleFacts/RelAlg/ITranslate.hs

```diff
@@ -489,7 +489,7 @@ insertTuples tblPair (Proj innerExp col heading skFn unqExp) db depth dpths
 
```

Comment:
Just as a general comment: we really liked the idea of evaluating sequents in models (i.e., the set of possible facts) as queries that were being evaluated in a database. However, our implementation of relational algebra became too complex and the major source of bugs in Razor. The underlying problem is that the columns of our tables were not named: for example, when joining two tables such as in `P(x, y) /\ Q(x)` the code looks at the index of the variables in common (in this case `x`) and joins table `P` with `Q` such that the value in the first column of `P` has to be equal to the first column of `Q`. Keeping track of relational expressions in this way may be a major source of confusion in the code.